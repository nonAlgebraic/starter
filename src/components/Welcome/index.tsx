import * as styles from './styles';

const Welcome = () => <div css={styles.container}>Hello!</div>;

export default Welcome;
