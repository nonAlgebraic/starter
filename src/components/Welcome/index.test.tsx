import { render } from '@testing-library/react';

import Welcome from './';

describe('Welcome', () => {
	it('renders', () => {
		const { container } = render(<Welcome />);

		expect(container).toMatchSnapshot();
	});
});
