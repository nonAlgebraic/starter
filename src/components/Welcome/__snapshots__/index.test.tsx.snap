// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders 1`] = `
.emotion-0 {
  color: red;
}

<div>
  <div
    class="emotion-0"
  >
    Hello!
  </div>
</div>
`;
